README.md

```diff
@@ -61,12 +61,23 @@ Run the production server from the bundled environment:
     bin/env bin/bundle exec bin/thin -p $PORT start
 
 ### Deploy with Docker
+
 To deploy PromDash with Docker, use the [prom/promdash](https://registry.hub.docker.com/u/prom/promdash/).
 By default, the image will start the [thin webserver](http://code.macournoyer.com/thin/)
 webserver in production mode. To run rake tasks like migrations, you
-can specify any kind of command as parameter to the Docker image:
+can specify any kind of command as parameter to the Docker image.  
+
+*Super easy quickstart deployment with docker*
+
+A quick start example, with a simple file based local db.
```

Comment:
Whole sentence please: `The following is a quick-start example for running PromDash with a file-based local database.`